src/gwt/src/org/rstudio/studio/client/projects/StudioClientProjectConstants_fr.properties

```diff
@@ -228,3 +228,5 @@ quartoBlogSubTitle=Créer un nouveau projet blog
 quartoBlogPageCaption=Créer un blog Quarto
 lblOrigin=Origine:
 useCondaenv=Utilisez condaenv avec les packages :
+generalTitle=General
+customProjectNameLabel=Project display name (defaults to folder name):
```

Comment:
Just a note for others that this string change will show up in the `locdiff` results since it is a new string.